proposals/0013-wave-size-range.md

```diff
@@ -125,16 +136,20 @@ These are where new or slightly altered errors are produced:
 
 Validation should confirm:
 
-* Each element in that is a power-of-two integer between 4 and 128 inclusive.
-  The third parameter may also be zero.
-* The minimum wave size is less than the maximum wave size.
-* The preferred wave size is zero or lies between the minimum and maximum.
-* Where validator version is less than 1.8, fail on shaders that use
- `RangedWaveSize`.
-* Validation should fail in an entry has both `RangedWaveSize` and `WaveSize`
- metadata.
-* Where validator version is greater than or equal to 1.8, fail when The tuple
- that the wave size range tag points to does not have exactly three elements.
+* The metadata `kDxilRangedWaveSizeTag` points to a tuple of exactly three
+ elements.
+* Each element of that tuple is a power-of-two integer between 4 and 128
+ inclusive or zero.
+* However, the first element (minimum wave size) is not zero.
+* The first element (minimum wave size) is less than the second
+ (maximum wave size).
+* The third element (preferred wave size) is greater or equal to the first
+ element (minimum wave size) and less than or equal to the second element
+ (maximum wave size) or zero.
+* Shaders with with versions less than 6.8 fail on shaders that use
```

Comment:
nit: double with, typo

Reply:
I might add, the wording is strange? Maybe a better way to say it is:
"Compilation with target shader model less than 6.8 will fail on shaders with the DxilWaveSize attribute".

Reply:
I agree the wording is awkward, but I have a couple disagreements with your proposed text. 

Compilation isn't what's happening here necessarily since this is about validation failures.

Part of the awkwardness might be because the bullet points are meant to refer to the header part that says "Validation should confirm".  So I expect this to be read "Validation should confirm (that) Shaders with versions less than 6.8 fail ..." I'll take another run at it though.